templates/components/cart/content.html

```diff
@@ -1,192 +1,135 @@
-                    {{/or}}
-                </div>
 
-                <div>
-                    <label for="qty-{{id}}">
-                        {{lang 'products.quantity'}}
-                    </label>
-                    
-                    <div>
-                        {{#if can_modify}}
-                            <button 
-                                class="c-button js-cart-update"
-                                data-cart-itemid="{{id}}"
-                                data-action="dec"
-                            >
-                                <span class="u-hidden-visually">
-                                    {{lang 'products.quantity_decrease' name=name}}
-                                </span>
-                                <svg>
-                                    <use href="#icon-keyboard-arrow-down" />
-                                </svg>
-                            </button>
-                        {{/if}}
-                        <input 
-                            class="js-form-input js-form-input-increment-total js-cart-item-qty-input"
-                            id="qty-{{id}}"
-                            name="qty-{{id}}"
-                            type="tel"
-                            value="{{quantity}}"
-                            data-quantity-min="{{min_purchase_quantity}}"
-                            data-quantity-max="{{max_purchase_quantity}}"
-                            data-quantity-min-error="{{lang 'products.quantity_min' quantity=min_purchase_quantity}}"
-                            data-quantity-max-error="{{lang 'products.quantity_max' quantity=max_purchase_quantity}}"
-                            min="1"
-                            pattern="[0-9]*"
-                            data-cart-itemid="{{id}}"
-                            data-action="manualQtyChange"
-                            aria-label="{{name}}"
-                            aria-live="polite"{{#unless can_modify}} disabled{{/unless}}>
-                        {{#if can_modify}}
-                            <button 
-                                class="c-button js-cart-update"
+                    <div class="c-cart__item-actions">
+                        <div class="c-cart__item-inputs">
+                            {{#if can_modify}}
+                                <button 
+                                    class="js-cart-update c-cart__item-qty-btn"
+                                    data-cart-itemid="{{id}}"
+                                    data-action="dec"
+                                >
+                                    <span class="u-hidden-visually">
+                                        {{lang 'products.quantity_decrease' name=name}}
+                                    </span>
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```

Comment:
Please use SVG sprite https://github.com/xfiveco/xcorner?tab=readme-ov-file#icons instead of hardcoding icons